lighthouse-core/test/gather/gatherers/seo/font-size-test.js

```diff
@@ -36,53 +60,249 @@ describe('Font size gatherer', () => {
     fontSizeGather = new FontSizeGather();
   });
 
-  it('returns information about font size\'s used on page', () => {
-    return fontSizeGather.afterPass({
-      driver: {
-        on() {},
-        off() {},
-        sendCommand(command, params) {
-          let result;
-          if (command === 'CSS.getComputedStyleForNode') {
-            if (params.nodeId === failingNode.nodeId) {
-              return Promise.reject();
-            }
-
-            result = {computedStyle: [
-              {name: 'font-size', value: params.nodeId === bodyNode.nodeId ? 10 : 20},
-            ]};
-          } else if (command === 'CSS.getMatchedStylesForNode') {
-            result = {
-              inlineStyle: null,
-              attributesStyle: null,
-              matchedCSSRules: [],
-              inherited: [],
-            };
+  it('returns information about font sizes used on page', async () => {
+    const driver = {
+      on() {},
+      off() {},
+      async sendCommand(command, params) {
+        let result;
+        if (command === 'CSS.getComputedStyleForNode') {
+          if (params.nodeId === failingNode.nodeId) {
+            throw new Error('This is the failing node');
           }
 
-          return Promise.resolve(result);
-        },
-        getNodesInDocument() {
-          return Promise.resolve(nodes);
-        },
+          result = {
```

Comment:
nit: can just return since nothing else is being done to `result` (IMO marginally more readable since you have to skip down pretty far to see it's just being returned, but, again, just a nit :)

Reply:
done